frontend/src/app/utils/gender.ts

```diff
@@ -0,0 +1,25 @@
+import { UNKNOWN } from "../../lang/en";
```

Comment:
I think we might be able to use some TS language features to make this file more robust. I put together a playground link here: https://www.typescriptlang.org/play?#code/C4TwDgpgBA4hB2ATCAnAks+wCWooF4AoKKAHygCIAzCAWwEMAbCC4syh51k8i4FevADOAdwD2DeN3Z8BwydN7wx8AEbZ49FCEWUxwABapdFFBCoBXIREQUA3ITYBjFUOBQAqgDkA0l4DyAOpeBJTefkFerIQA9DFQACrgEEJOKNhg7ggWtEJQWtAA1hqI9FDAqOmqzFAAFAbAwGBCAFxxyABuAHTAYjHYHYIAXrQAjABMAMwxiCrAALRWEPPZufMa86CQqemZm0Yg8wXzHaiHpfAA5qhiVvMArADsqgYAlLHxgohQaADk34wxKdymIoNZoFYoLQgdB6CJ6CAoFQUBJykZaGCNE5oGg0RpCmiIIiGIVoFQxCgoMwhHktsshPQaFAXMgulAAGIUvF5Jz0awAGh+ePgBI+UBE0F58CgS3yzNc7kQ2CcOBUWkRGjcEHo3yEWOgIgM9HcEt+ZigZiYjERsw0lyg2DyDDAYDthNgCGQ6EwODwdNpoIMOUE80tpWq0Dc6SuQkccXdPnKyXlwn49A07g07tJiKM5pE2EYjCh9FJYIs5sEiMswAr0CVQjAjARUEGjAsKXyjCEoPgEBsIKgqlhiGQ3167rgSFQGAQvsRYrp+SQreV0FOKAZhBcqY90+9c9wIAAIo6mwiWlAAN5QADaiazU69s6wR4Aupeo26AL6hK9sGhOAgS8KHZOgmBYfk2CAkCAFkIIoKCSDTYRxEkECEjkPJANoaAuRQ0QJEERC2AI9DKEwwQnT5JwLEYDQ8MpMjiKQqBlDUDR1RArwVHUTRtCgLlrn3NiVHmHdyRQWg7RIkh9DzED-EMYxWLMSxrEQECAAU1NQUT3AnKiJRQEjfz5FM3AcYgdzcJEKQYYAn1QfwqAAZQgAAPCwmC0rRgD7TdORQU9G2bRF8DqNhhK9VowX4O1bzfQhXk-eKrgIAA+a82HjWZ4F+dw+wHCcnCMJwCUkqA6EyREtAEEA8j1eBsSRQsKkpeJnUE6VqrwOqETyAsiygAArKx3DMWsUGlQQqtoGr8hQeq2GYdxotQIQQvPcKoHWzc2BIeMpRlaxWsYdqk0gNi6rEAt0onCphuU+V5sLPSKAOd1biaCx3DEKhuTOi6huLYdlyq+AcmkQ74gaJpWjiNx6HKmEUCoQERC6FxaBiABHDs3GwVwYgAFkmAAGUZyceSZHhieFDl6TYICLTZkh2DIFmx11mBQI4loReYwBRXo6XmKg2tQUM6BhIRxNQYB03gNntnFlESAOqAugl87UFqWo9pS3bPT0x092fH0j0y7KSFti0ICm6V-FUEaIBVLocyEWonIPV9QC2sLXi6LF22QL3Nbtvb8jyH2X3nCOoFeBw7e-d47a6Z19cuV5rdjy3-bPMLb0uN8k7YSaK2lPbNsLhEumYK5DCgLLyagAB+Y39xr0K65GsQNFqRDKBzy9wgCYIHG-KybIMlJ3AiihlEzKuTcpbB850RwbLEZh67ES5akkhyfZc9yvJ8vyAqEIKA4RWpbwqNxBWocCuGfofZCohRn57XDykqbAEZWxMA7BQUuOd4ygVfhAQUlFhAllSHRBigkmJYQUFAbcrgd4QD3gfI+xoT5uU8t5RgvkUD+Q2jfWuIB77gKgJAsekQKAYKAA

Let me know if you want to chat through anything since there are a couple TS nerdery things that I pulled together to make the above work

Reply:
@fzhao99 I like that idea of using a typed dictionary for the gender values (non-blocking if we wanna have that in a separate PR). Saw [this comment too](https://github.com/CDCgov/prime-simplereport/pull/7161/files#r1455697813) which makes me wonder if there's any particular reason the test result details are not currently translated?

Reply:
OOO thank you for nerding out! love that -- will take a look now! 👀